docs/architecture-decisions/0005-implement-an-accessible-navigation-system.md

```diff
@@ -0,0 +1,214 @@
+# 5. Implement an accessible navigation system
+
+Date: 2024-03-22
+
+## Status
+
+Accepted
+
+## Context
+
+Our initial releases of this theme used the [SmartMenus](https://www.smartmenus.org/) library to build its
+navigation menus. While this library is easy to implement and pretty useful, its
+output was flagged during usability and accessibility testing as needing some
+improvements.
+
+The markup which the user finally interacts with looks like this:
+
+```html
+<nav aria-label="Under the Lens: Women Biologists and Chemists at MIT (1865-2024)">
+  <ul class="navigation sm sm-mint" data-smartmenus-id="17111193231069111">
+    ...
+    <li>
+        <a href="/s/under-the-lens/page/timeline" class="has-submenu" id="sm-17111193231069111-1" aria-haspopup="true" aria-controls="sm-17111193231069111-2" aria-expanded="false">Timeline
+        	<span class="sub-arrow"></span>
+        </a>
+        <ul id="sm-17111193231069111-2" role="group" aria-hidden="true" aria-labelledby="sm-17111193231069111-1" aria-expanded="false" style="width: auto; min-width: 10em; display: none; max-width: 20em; top: auto; left: 0px; margin-left: 0px; margin-top: 0px;" class="sm-nowrap">
+            <li>
+                <a href="/s/under-the-lens/page/timeline1">Timeline: The Beginnings</a>
+            </li>
+            ...
+        </ul>
+    </li>
+    ...
+  </ul>
+</nav>
+```
+
+The issue is that the element triggering the flyout menu (the 
+`<span class="sub-arrow"></span>` tag) is _inside_ the anchor element, making it
+challenging (if not impossible) for users of assistive technologies to separate
+their interactions with the menu link and the submenu toggle.
+
+The current release of SmartMenus (1.2.1) does not support placing this toggle
+element outside of the link element, so it will be necessary to replace it with
+a library which does support this behavior.
+
+
+### Option 1: SmartMenus 2
+
+The maintainer for SmartMenus is aware of this problem in the library, as we and
+others have reported it via GitHub (see [issue 204](https://github.com/vadikom/smartmenus/issues/204) and [issue 231](https://github.com/vadikom/smartmenus/issues/231)). He has
+committed to including this as an improvement in the future release of
+SmartMenus 2, and the documentation already available [includes this among its available
+features](https://configurator.smartmenus.org/).
+
+However, at the time of writing [SmartMenus 2 is only available via one alpha
+release](https://github.com/vadikom/smartmenus/releases) - and the maintainer has [cautioned us against using it in a production
```

Comment:
If the frequency of past version tags are any indication, we could be waiting a long time for anything beyond alpha...